### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by javadr**
In this [line](https://github.com/AllenDowney/LittleBookOfSemaphores/blob/2d71f16952c88baa69f5dbc8dd6dc1fc9c8b4550/book/book.tex#L3607), `barbers` seems to be `barber`

### CODE REVIEW

**Comment by AllenDowney** [comment]
Should it be "the barbers"?

**Comment by javadr** [comment]
As you better know than me, in the original problem by Dijkstra there is only one barber like in your book. 
Therefore, `the barber` would be better, I think.

**Comment by AllenDowney** [comment]
Sounds good. Let me know when you are done and I will merge. Thank you!

**Comment by javadr** [comment]
Done!

File: book/book.tex [hunk 1]
```diff
@@ -3806,7 +3807,7 @@ \subsection{FIFO barbershop solution}
 This solution is in \verb"sync_code/barber3.py" (see~\ref{sync.py}).
 
 The only way I can think of to solve this problem is to create a third
-kind of thread, which I can an usher.  The ushers manage {\tt queue1}
+kind of thread, which I can name it an usher.  The ushers manage {\tt queue1}
```
Comment (AllenDowney) [inline book/book.tex:3809]: Can I ask you to change this to "which I'll call an usher"?

Or I can merge this and then make the change myself.

**Review by AllenDowney — CHANGES_REQUESTED** [review]

File: book/book.tex (hunk 1 above)
Comment (javadr) [reply book/book.tex]: Sounds better. 
Done.

**Comment by AllenDowney** [comment]
Thank you very much!

**Pull request merged**